data/amd/register/vmcb/000h.yml

```diff
@@ -0,0 +1,37 @@
+- name: 000h
+  long_name: "000h (vector 0)"
+  purpose: |
+      "
+      Intercept reads of CR0–15, respectively
+      "
+  size: 32
+  arch: amd64
+
+  access_mechanisms:
+      - name: read
+        offset: 000h
+        component: vmcb
+
+      - name: write
+        offset: 000h
+        component: vmcb
+        
+  fieldsets:
+    name: latest
+    condition: "Fieldset valid on latest version of the AMD architecture"
+    size: 32
+
+    fields:
+      - name: "Bits 15_0"
+        long_name: "Intercept reads of CR0–15, respectively" 
+        lsb: 0
+        msb: 15
+        readable: True
+        writable: True
+
+      - name: "Bits 31_16"
+        long_name: "Intercept reads of CR0–15, respectively" 
```

Comment:
Just noticed this too: the manual describes bits 31:16 as "Intercept writes of CR0–15, respectively" (i.e. s/reads/writes)